tests/e2e/support/objects/app-files/spaces/actions.ts

```diff
@@ -158,3 +160,38 @@ export const changeQuota = async (args: {
 
   await sidebar.close({ page: page })
 }
+
+/**/
+
+export const changeSpaceImage = async (args: {
+  id: string
+  page: Page
+  resource: File
+}): Promise<void> => {
+  const { id, page, resource } = args
+  await sidebar.open({ page: page })
+  await sidebar.openPanel({ page: page, name: 'space-actions' })
+
+  const [fileChooser] = await Promise.all([
+    page.waitForEvent('filechooser'),
+    page.locator(editImageOptionSelector).click()
+  ])
+
+  await Promise.all([
+    page.waitForResponse(
+      (resp) =>
+        resp.url().endsWith(encodeURIComponent(id)) &&
+        resp.status() === 200 &&
+        resp.request().method() === 'PATCH'
+    ),
+    page.waitForResponse(
+      (resp) =>
+        resp.url().endsWith(resource.name) &&
+        resp.status() === 207 &&
+        resp.request().method() === 'PROPFIND'
+    ),
+    fileChooser.setFiles(resource.path)
+  ])
+
+  await sidebar.close({ page: page })
```

Comment:
Could we also check that the `href` of the `img` embedded into the project space root view changed accordingly?

Reply:
@SwikritiT maybe could you please give me advise how do it better?

![Screenshot 2022-06-20 at 20 29 16](https://user-images.githubusercontent.com/84779829/174661192-3c54b655-0118-4904-acf9-e329c205b44d.png)


check that class contains src atribute, but I need first get image_id 
or more easer: that class ".space-overview-image" appears

```
const locator = await page.locator('.space-overview-image')
expect(locator).toHaveAttribute('src', 'blob:https://host.docker.internal:9200/ebf3d7fd-da5c-4cbb-9216-132c8b999e1d')
```

Reply:
I think checking that the class contains the `src` attribute is better
I could get the values of the src attribute like this
```  
const img = await page.locator('.space-overview-image')
const src = await img.evaluate((e) => (e as HTMLImageElement).src)
console.log(src)
```
Maybe you can assert that it's not empty? In case of update store the previous `src` and assert it with new, confirming that they aren't the same? 
Does this make sense or will be of any help to you?

Reply:
Thanks for the suggestion: 
```
const img = await page.locator('.space-overview-image')
const src = await img.evaluate((e) => (e as HTMLImageElement).src)
expect(src).not.toBe(spaceImageId)
spaceImageId = src
```

first line checks that image exist on web
3 line checks that src ≠ '' in first image setting and src ≠ old spaceImageId  subsequent image setting